packages/react-changelogs/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const webpack = {
+  entry: './src/index.js',
+  output: {
+    filename: 'target/bundle.js',
+  },
+  devtool: 'source-map',
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /.tsx?$/,
+        loader: 'babel-loader!ts-loader',
```

Comment:
We shouldn't need the ts loader here if we have babel configured to use the typescript preset.